xl/plugins.py

```diff
@@ -144,6 +144,8 @@ def uninstall_plugin(self, pluginname):
         return False
 
     def enable_plugin(self, pluginname):
+        if pluginname == 'lyricsviewer':
```

Comment:
Let's make it fail with an exception then. I wanted to have no exception printed to output/logfile in a case like this when the behaviour was expected, but I don't care much.

PS: I pushed a new commit, which is identical to the old one except for missing these two lines.